### DISCUSSION TITLE
Is SECAM correct?

### DISCUSSION THREAD

**Discussion by andrew-davie** [stella-emu/stella#881] [Q&A] [answered]
I am wondering if SECAM emulation is behaving correctly.
When I switch a ROM to "SECAM" TV-type, the visible screen shifts to the top of frame.  I'm trying to understand why this is so, and if this is also what would happen on an actual SECAM unit/TV. Ta.

**Comment by thrust26** [comment, marked as answer]
I suppose you are switch an NTSC game to SECAM, right? That switches emulation from 60Hz to 50Hz, the latter having ~50 more scanlines. This means each scanline uses less vertical space. And that causes the picture to move up. The same happens when you switch from NTSC to PAL.

This cannot happen on a real TV, because you cannot switch TV modes there.

**Comment by andrew-davie** [comment]
OK, TY for the reply.
